modules/openxr/extensions/openxr_eye_gaze_interaction.cpp

```diff
@@ -74,11 +77,16 @@ bool OpenXREyeGazeInteractionExtension::supports_eye_gaze_interaction() {
 	// The extension being available only means that the OpenXR Runtime supports the extension.
 	// The `supportsEyeGazeInteraction` is set to true if the device also supports this.
 	// Thus both need to be true.
-	return available && properties.supportsEyeGazeInteraction;
+	// In addition, on mobile runtimes, the proper permission needs to be granted.
+	if (available && properties.supportsEyeGazeInteraction) {
+		return !OS::get_singleton()->has_feature("mobile") || OS::get_singleton()->has_feature("PERMISSION_XR_EXT_eye_gaze_interaction");
```

Comment:
Shouldn't it be calling `supportsFeature` for the permission, or is there something that eventually calls that as part of `has_feature` on mobile?

Reply:
Yeah, `OS_Android` relays the `has_feature` call to the java layer which invokes `supportsFeature` on the registered plugins.